bundles/org.openhab.binding.mybmw/src/main/resources/OH-INF/update/thing-update.xml

```diff
@@ -4,87 +4,73 @@
 	xsi:schemaLocation="https://openhab.org/schemas/update-description/v1.0.0 https://openhab.org/schemas/update-description-1.0.0.xsd">
 
 	<thing-type uid="mybmw:bev_rex">
-		<instruction-set targetVersion="1">
+		<instruction-set targetVersion="2">
 			<!-- channels to be removed -->
-			<remove-channel id="charge-info" groupIds="status"/>
-			<remove-channel id="motion" groupIds="status"/>
 			<!-- channels to be added -->
-			<add-channel id="charge-remaining" groupIds="status">
-				<type>mybmw:charging-remaining-channel</type>
-				<label>Remaining Charging Time</label>
+			<add-channel id="state-update" groupIds="update">
+				<type>mybmw:state-update-channel</type>
+				<label>Force state update channel</label>
 			</add-channel>
-			<add-channel id="last-fetched" groupIds="status">
-				<type>mybmw:last-fetched-channel</type>
-				<label>Last Openhab Update Timestamp</label>
+			<add-channel id="charging-update" groupIds="update">
+				<type>mybmw:charging-update-channel</type>
+				<label>Force charging update channel</label>
 			</add-channel>
-			<add-channel id="estimated-fuel-l-100km" groupIds="range">
-				<type>mybmw:estimated-fuel-l-100km-channel</type>
-				<label>Estimated Fuel Consumption l/100km</label>
-			</add-channel>
-			<add-channel id="estimated-fuel-mpg" groupIds="range">
-				<type>mybmw:estimated-fuel-mpg-channel</type>
-				<label>Estimated Fuel Consumption mpg</label>
+			<add-channel id="image-update" groupIds="update">
+				<type>mybmw:image-update-channel</type>
+				<label>Force image update channel</label>
 			</add-channel>
 			<!-- channels to be updated -->
 		</instruction-set>
 	</thing-type>
 	<thing-type uid="mybmw:bev">
-		<instruction-set targetVersion="1">
+		<instruction-set targetVersion="2">
 			<!-- channels to be removed -->
-			<remove-channel id="charge-info" groupIds="status"/>
-			<remove-channel id="motion" groupIds="status"/>
 			<!-- channels to be added -->
-			<add-channel id="charge-remaining" groupIds="status">
-				<type>mybmw:charging-remaining-channel</type>
-				<label>Remaining Charging Time</label>
+			<add-channel id="state-update" groupIds="update">
+				<type>mybmw:state-update-channel</type>
+				<label>Force state update channel</label>
+			</add-channel>
+			<add-channel id="charging-update" groupIds="update">
+				<type>mybmw:charging-update-channel</type>
+				<label>Force charging update channel</label>
 			</add-channel>
-			<add-channel id="last-fetched" groupIds="status">
-				<type>mybmw:last-fetched-channel</type>
-				<label>Last Openhab Update Timestamp</label>
+			<add-channel id="image-update" groupIds="update">
+				<type>mybmw:image-update-channel</type>
+				<label>Force image update channel</label>
 			</add-channel>
 			<!-- channels to be updated -->
 		</instruction-set>
 	</thing-type>
 	<thing-type uid="mybmw:conv">
-		<instruction-set targetVersion="1">
```

Comment:
Don't remove the instructions for V1, previous instructions may never be deleted, only additions are allowed.

Reply:
OK I wasn't aware about this. Re-added. Just for completeness: should I add the new channels in the distro update file?

Reply:
That is not needed, instructions are fine now.